tapir/settings.py

```diff
@@ -198,9 +201,9 @@
 
 WEASYPRINT_BASEURL = "/"
 
-REG_PERSON_BASE_DN = "ou=people,dc=supercoop,dc=de"
+REG_PERSON_BASE_DN = env("REG_PERSON_BASE_DN", default="ou=people,dc=supercoop,dc=de")
```

Comment:
Same here, if this does not differ between envs, I feel it's probably better to keep it in settings.py for now.

Reply:
OK, if we go for a everyone-should-fork-strategy, it makes perfect sense without the env-var.